go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -437,6 +441,31 @@ func (as *ovnAddressSets) AddIPs(ips []net.IP) error {
 	return nil
 }
 
+// TEMP
+func (as *ovnAddressSets) AddIPsReturnsOps(ips []net.IP) ([]ovsdb.Operation, error) {
```

Comment:
@tssurya you could have `AddIPs()` call this function and then run `TransactAndCheck()` on the returned ops

Reply:
oh yes, nice that we don't have to have duplicated things.